### ISSUE TITLE
[PL][Warlock] U wszystkich Petów HP jest o ok. 50% zawyżone.

### ISSUE THREAD

**Issue by BialyADI** [Atlantiss/BugTracker#7933]
Jak jest:
Po jakiejś z ostatnich poprawek HP Petów Warlocka wzrosło ok 50%.
Na moim Warlocku Felhunter(pies) miał 70k HP teraz ma 115k, sporo więcej niż ja sam bez xD.
Jakiś ostatni UpData to musiał popsuć.

Jak powinno być:
Pety powinny się skalować z Panem:
Stamina 75%
Intellect 30%
Armor 35%
Resistances 40%
Spell Damage (highest bonus) 15% (as spell damage)
57% (as attack power)
Spell Penetration 100%
Resilience 100%

Teraz ciężko ocenić jak jest dokładnie. Ale HP jest o ok 50% wyższe niż było, nie wiem jak jest z DMG może tez jest o 50% przez złe skalowanie z statystykami Pana.
http://wow.gamepedia.com/Warlock_minion
http://et.worldofwarcraft.wikia.com/wiki/Minion

**Comment by sigmundfreud2016** [comment]
Yup, lock pets have incredible amount of HP on atlantiss, especially felguard (200k+).

**Comment by BialyADI** [comment]
Pety Hunta jednak maja podobnie, przyjrzałem się temu jak zobaczyłem Pet Hunta na BG miał 150k hp ehh. https://github.com/Atlantiss/BugTracker/issues/7954.

**Comment by Jenovas** [comment]
" nie wiem jak jest z DMG może tez jest o 50% przez złe skalowanie z statystykami Pana." - jak nie wiesz to nie pisz. Nikt tego nie będzie sprawdzał.

**Comment by BialyADI** [comment]
Wiem że wszyscy maja wyjebane jajka na to, ja tylko napisałem to co zauważyłem biegając na Bg czy Arenie ^^.
Sporo Lockow i Huntow graja teraz w Demo lub BM, pewno nie dla tego że Pet słabiej bije i ma mniej HP.
Gdybym miał jak sprawdzić co się zle skaluje w Petach bym się chwilę pobawił i sprawdził.

**Comment by BialyADI** [comment]
Bump ;x.
Warlocki z Soul Link i Petami z tą ilością HP są unkillabla.

**Comment by TranceAccount** [comment]
Your title is wrong.
>On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player.

**Comment by BialyADI** [comment]
bump

**Comment by TranceAccount** [comment]
Your title is still wrong.

**Comment by BialyADI** [comment]
Jest tu ktoś, od miesiąca pety mają więcej HP od właścicieli ;D.

**Comment by Jenovas** [comment]
I będą miały jeszcze jakiś czas, są ważniejsze rzeczy obecnie do poprawy.

**Comment by kadi2** [comment]
Pety nie scaluja sie tylko i wylacznie z 75% staminy wlasciciela. Po co tworzysz report i wrzucasz do niego na sile invalid proof?

**Comment by BialyADI** [comment]
Masz dowód i na filmiku i na wiki, U Huntów jeszcze jest ok.
Ale u Lockow jest to przegięte teraz że pet ma więcej HP niż on i jeszcze jest jego "baterią" dzięki Soul Link.

A co do skalowania powinien się skalować z 75% staminy właściciela i każda klasa ma też jakieś talenty by Pet miał więcej HP.

**Comment by kadi2** [comment]
https://github.com/Atlantiss/BugTracker/issues/7027#issuecomment-206474818 There is a proof about that it shouldn't scale just with 75% of owner's hp as you stated in the first post.

**Comment by BialyADI** [comment]
Bump. HP petów nie poprawnie się skaluje.

**Comment by TranceAccount** [comment]
Bump. Your title is still wrong.

**Comment by Fearyourself** [comment]
Bump. Fix warlock pets health jesus christ.

**Comment by Fearyourself** [comment]
Petition to reopen??

**Comment by Piklo** [comment]
Its closed?

**Comment by Fearyourself** [comment]
Well open it the hell up and fix warlock minion's health already. Hp scaling way too low.

**Comment by Fearyourself** [comment]
I read that wrong. It's obviously been given up on though.
